src/intercooler.js

```diff
@@ -72,9 +72,10 @@ var Intercooler = Intercooler || (function() {
   }
 
   function hideIndicator(elt) {
-    if (elt.data('ic-use-transition')) {
+    if (elt.data('ic-use-transition') || elt.data('is-indicator-cleared')) {
```

Comment:
Should this be `ic-` instead of `is-`?

Reply:
For data (rather than attr()) storage, using the ic- prefix is fine IMO.

Reply:
Fixed.

Reply:
Thanks.